python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -599,6 +605,186 @@ async def verify_calib(self, image_type, job_id_calib, exposure_ids_dict):
 
         return response
 
+    async def check_verification_stats(self, image_type, job_id_calib, job_id_verify):
+        """Check verification statistics.
+
+        Parameters
+        ----------
+        image_type:
+            Image type. Verification currently only implemented for ["BIAS",
+            "DARK", "FLAT"].
+
+        jod_id_calib : `str`
+            Job ID returned by OCPS during previous calibration
+            generation pipetask call.
+
+        job_idverify : `str`
+            Job ID returned by OCPS during previous calibration
+            verification pipetask call.
+
+        Returns
+        -------
+        certify_calib : `bool`
+            Booolean indicating if the calibration should be certified or not.
+
+        numStatErrors : `dict`[`str`][`str`]
+            Dictionary with the total number of tests failed per exposure and
+            per cp_verify test type. If there are not any tests that failed,
+            `None` will be returned.
+
+        thresholds : `dict`[`str`][`int`]
+            Dictionary reporting the different thresholds used to decide
+            whether a calibration should be certified or not (see `Notes`
+            below). If there are not any tests that failed,
+            `None` will be returned.
```

Comment:
See my comment above. 

If you are returning `None`, please, update the description to reflect that, 

```
thresholds : `dict`[`str`][`int`] or `None`
```

Although, I would advocate against returning `None` in case of success. I'd rather  return a dictionary indicating that 0 tests failed.